molecule/default/Dockerfile.j2

```diff
@@ -13,11 +13,21 @@ rm -f /lib/systemd/system/sockets.target.wants/*initctl*; \
 rm -f /lib/systemd/system/basic.target.wants/*;\
 rm -f /lib/systemd/system/anaconda.target.wants/*;
 
-RUN if [ $(command -v apt-get) ]; then apt-get update && apt-get upgrade -y && apt-get install -y python sudo bash ca-certificates && apt-get clean; \
-    elif [ $(command -v dnf) ]; then dnf makecache && dnf --assumeyes install python sudo python-devel python2-dnf bash which selinux-policy firewalld jemalloc && dnf clean all; \
-    elif [ $(command -v yum) ]; then yum makecache fast && yum update -y && yum install -y python sudo yum-plugin-ovl bash which selinux-policy firewalld jemalloc&& sed -i 's/plugins=0/plugins=1/g' /etc/yum.conf && yum clean all; \
-    elif [ $(command -v zypper) ]; then zypper refresh && zypper update -y && zypper install -y python sudo bash python-xml && zypper clean -a; \
-    elif [ $(command -v apk) ]; then apk update && apk add --no-cache python sudo bash ca-certificates; fi
+# CentOS
+RUN if [[ $(grep CentOS /etc/redhat-release) ]]; then \
+  yum install -y epel-release ;\
```

Comment:
To install Redis and Python 3, both RHEL and CentOS require EPEL
Should I move it to pulp3-redis, add external role, or leave it as is?